specifyweb/frontend/js_src/lib/components/QueryBuilder/ToForms.tsx

```diff
@@ -34,10 +34,19 @@ export function QueryToForms({
       ? (results[index]![queryIdField] as number)
       : Array.from(selectedRows)[index];
 
+  const containUndefinedResults = React.useMemo(
+    () => results.includes(undefined),
+    [results]
+  );
+
```

Comment:
the code looks good (and good job using .includes - even better than .some if that works for you), but I am confused on what you are trying to achieve here.

the code in present form disables the "Browse in Forms" if any result is not fetched yet - is that what you are trying to do?
i.e if query has 1000 results, the "Browse in Forms" will remain disabled until the user scrolls to the very bottom of the query results table (and wait for it to load) - only then would "containUndefinedResults" become false - surely this is not what you are going for

Reply:
no only when the first 80 are loaded

Reply:
and it allows to not have the browse in form when there is no results

Reply:
see me break it:

1. go to the last result
2. the "Browse in Forms" becomes disabled


https://github.com/specify/specify7/assets/40512816/e4286034-4574-48aa-acca-9dc1650003bd

Reply:
When query returned to results, the `results` array looks like this:

![Screenshot 2023-10-11 at 19 23 02](https://github.com/specify/specify7/assets/40512816/f30ece3d-5697-466a-b831-c8c6209bc365)

take a look at why `undefined` is there, and make it not be there (and check if that change is safe to do - i.e double check it doesn't break anything else) - then there won't be need for `containsUndefined`